scripts/create-new-version

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+echo "Welcome to the guided process of updating the Guides search index"
+echo "We will now be running a mix of automated (🤖) and manual (👩‍💻) steps, please read the prompts carefully."
+
+STARTING_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+TEMP_BRANCH="create-new-guides-version-$(( $RANDOM % 1000 ))"
```

Comment:
Instead of `$RANDOM % 1000` (which creates a chance of name collision over time), is it possible to use UNIX timestamp (current time)?